integration_tests/test_stac.py

```diff
@@ -707,6 +707,38 @@ def test_stac_collection(stac_client: FlaskClient):
     validate_items(_iter_items_across_pages(stac_client, item_links), expect_count=306)
 
 
+# TODO
+# We probably should check the conformance classes being returned.
+# They're in the root /stac/ response under the `conformsTo` item.
+# They are also served up at /stac/conformance in recent releases
+
+#######
+# Tests for STAC API - Features/Collections
+# https://api.stacspec.org/v1.0.0/ogcapi-features/
+# https://github.com/radiantearth/stac-api-spec/tree/release/v1.0.0/ogcapi-features
+#
+# These cover the responses available at /stac/collections/*
+# The spec offers either a cut down version called 'Collections', but Explorer
+# supports the full 'Features' specification.
+
+
+@pytest.mark.parametrize("env_name", ("default",), indirect=True)
```

Comment:
Since this is a performance issue, shouldn't this also be enabled for the postgis backend?

Reply:
We're not using the PostGIS backend yet, and, there's a LOT of integration tests here that aren't testing against the PostGIS index either. I was afraid I'd fall down another rabbit hole if I started expanding the test coverage.

The table structure and query structure are identical between the two, with the exception of the naming of the String `name` field that is used to connect the cubedash product table with the odc product/agdc dataset_type table.